challenges-22-plus.yml

```
@@ -0,0 +1,192 @@
+    
+                  <p>Here are some usefull contact numbers in case you are in an emergency and
+                  in need of help!</p>
+
+                  <p><strong>Hope for Paws</strong>: +1 310-880-1416</p>
+
+                  <p><strong>National Mill Dog Rescue</strong>: +1 719-683-3334</p>
+
+                  <p><strong>Catskill Animal Sanctuary</strong>: +1 845-336-8447</p>
+              </turbo-frame>
+          {% endblock %}
+        c: |
+          {% block body %}
+              <h1>Useful Contact Numbers</h1>
+
+              <p>Here are some usefull contact numbers in case you are in an emergency and
+              in need of help!</p>
+
+              <turbo-frame src="{{ path('contact_numbers') }}">
+                  <p><strong>Hope for Paws</strong>: +1 310-880-1416</p>
+
+                  <p><strong>National Mill Dog Rescue</strong>: +1 719-683-3334</p>
+
+                  <p><strong>Catskill Animal Sanctuary</strong>: +1 845-336-8447</p>
+              </turbo-frame>
+          {% endblock %}
+        d: |
+          {% block body %}
+              <h1>Useful Contact Numbers</h1>
+
+              <p>Here are some usefull contact numbers in case you are in an emergency and
+              in need of help!</p>
+
+              <turbo-frame>
+                  <p><strong>Hope for Paws</strong>: +1 310-880-1416</p>
+
+                  <p><strong>National Mill Dog Rescue</strong>: +1 719-683-3334</p>
+
+                  <p><strong>Catskill Animal Sanctuary</strong>: +1 845-336-8447</p>
+              </turbo-frame>
+          {% endblock %}
+      correct: a
+  
+lazy_frame_full_frame_2:
+  question: |
+    Based on the last question, if I wanted my footer frame to load lazily, only when
+    the user scrolls down to the footer of my page, how can I achieve this?
+  explanation: |
+    If a frame you need to render is too heavy and uses many resources to load, you can
+    economize a bit of resources by adding `loading="lazy"` on `<turbo-frame>` element.
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          ```html
+          <turbo-frame id="contact-numbers" src="{{ path('contact_numbers') }}" data-turbo-frame="lazy"></turbo-frame>
+          ```
+        b: |
+          ```html
+          <turbo-frame id="contact-numbers" src="{{ path('contact_numbers') }}" data-turbo-load="lazy"></turbo-frame>
+          ```
+        c: |
+          ```html
+          <turbo-frame id="contact-numbers" src="{{ path('contact_numbers') }}" loading="lazy"></turbo-frame>
+          ```
+        d: |
+          There is no such feature in Turbo Frames. All of your frames will load as soon as
+          the `<turbo-frame>` elements are ready in the DOM.
+      correct: c
+
+frame_target:
+  question: |
+    When we click on a link inside a Turbo Frame, Turbo will look for a reference to that frame in our
+    response HTML so that it can replace only that part of our page. But if our link takes us to a
```

```suggestion
    response HTML so that it can use only that part of the new page. But if our link takes us to a
```

There may be a better way to word it. But before, it sounded like it would look for the frame in the new HTML "in order to" figure out which part of the *current* page to replace (which I know isn't what you meant).